### ISSUE TITLE
"ValueError: too many values to unpack" when trying to identify spammers using "spammer channel id/link" method

### ISSUE THREAD

**Issue by MDP43140** [ThioJoe/YouTube-Spammer-Purge#185]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

I was trying to report some sex spammer on some YouTube channel. it should continue but then the ValueError shows up

### Release version

v2.2.5

### Steps to reproduce

1. run YouTubeSpammerPurge.
2. Select "Scan recent videos for a channel".
3. Enter channel ID (in my case was /user/miawaug), and recent videos to be scanned (like usual).
4. Select "Enter Spammer's channel ID(s) or link(s)".
5. Put the spammer ID/Link (which in my case was UCZsYCXz3yF5BxFwzeMzyKBQ or https://youtube.com/channel/UCZsYCXz3yF5BxFwzeMzyKBQ).
6. Bam, ValueError...

### What platform are you seeing this problem on?

Linux

### Relevant log output

```shell
mdp43140@linux:~ python3 ./YouTubeSpammerPurge.py
[...]
---------- Scanning Options --------------------------------------
[...]
Choice (1-6): 2

Enter a channel ID or Link to scan recent videos from
   > If scanning your own channel, just hit Enter

Enter Here: https://www.youtube.com/user/miawaug 

Chosen Channel: MiawAug

Enter the number most recent videos to scan back-to-back (up to 5):

Number of Recent Videos (1-5): 5

[...]
-------------------------------------------------------
~~~~~~~~~~~ Choose how to identify spammers ~~~~~~~~~~~
-------------------------------------------------------
 1. (RECOMMENDED): Auto-Smart Mode: Automatically detects multiple spammer techniques
 2. Sensitive-Smart Mode: Much more likely to catch all spammers, but with significantly more false positives
 3. Enter Spammer's channel ID(s) or link(s)
 4. Scan usernames for criteria you choose
 5. Scan comment text for criteria you choose
 6. Scan both usernames and comment text for criteria you choose
 7. ASCII Mode: Scan usernames for ANY non-ASCII special characters (May cause collateral damage!)
 Note: With 'Not Your Channel Mode' enabled, you can only report matched comments while using 'Auto-Smart Mode'.

Choice (1-7): 3
Enter the  Channel link(s) or ID(s) of the spammer (comma separated): https://youtube.com/channel/UCZsYCXz3yF5BxFwzeMzyKBQ
Traceback (most recent call last):
  File "./YouTubeSpammerPurge.py", line 2776, in <module>
    main()
  File "./YouTubeSpammerPurge.py", line 2415, in main
    filterSettings = prepare_filter_mode_ID(currentUser, scanMode, config)
  File "./YouTubeSpammerPurge.py", line 1736, in prepare_filter_mode_ID
    processResult = process_spammer_ids(inputtedSpammerChannelID)
  File "./YouTubeSpammerPurge.py", line 1136, in process_spammer_ids
    valid, IDList[i] = validate_channel_id(inputList[i])
ValueError: too many values to unpack (expected 2)
```

**Comment by TechStudent11** [comment]
I think the program expected two scammers. That shouldn't be right.

**Comment by ThioJoe** [comment]
Hm the problem is with the channel validation function. I won't be able to work on it until later today. Does it work if you put in the channel Id exactly, and not as a link?

**Comment by MDP43140** [comment]
> Hm the problem is with the channel validation function. I won't be able to work on it until later today. Does it work if you put in the channel Id exactly, and not as a link?

nope doesnt even work, channel id (UCZsYCXz3yF5BxFwzeMzyKBQ) and url (https://youtube.com/channel/UCZsYCXz3yF5BxFwzeMzyKBQ) wont work

**Comment by ThioJoe** [comment]
Should be fixed in latest release, 2.2.6
